package.json

```diff
@@ -14,17 +14,17 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.21.5",
-    "@bcrs-shared-components/base-address": "2.0.7",
+    "@bcrs-shared-components/base-address": "2.0.9",
     "@bcrs-shared-components/breadcrumb": "2.1.11",
     "@bcrs-shared-components/confirm-dialog": "1.2.1",
     "@bcrs-shared-components/corp-type-module": "1.0.11",
     "@bcrs-shared-components/court-order-poa": "2.1.4",
     "@bcrs-shared-components/document-delivery": "1.2.1",
-    "@bcrs-shared-components/enums": "1.0.50",
+    "@bcrs-shared-components/enums": "1.0.51",
     "@bcrs-shared-components/expandable-help": "^1.0.0",
     "@bcrs-shared-components/folio-number-input": "1.1.18",
-    "@bcrs-shared-components/interfaces": "1.0.75",
-    "@bcrs-shared-components/mixins": "1.1.25",
+    "@bcrs-shared-components/interfaces": "1.0.76",
+    "@bcrs-shared-components/mixins": "1.1.27",
```

Comment:
Didn't some of these jump by 2?

Reply:
Only @bcrs-shared-components/base-address